### PR TITLE
package.json: add lint-plugin script

### PR DESCRIPTION
**Pull request by suomiy**
Faster and easier linting for plugin development. Usage:

`yarn lintme packages/kubevirt-plugin`

- this was created as reaction to #1826

### CODE REVIEW

**Comment by mareklibra** [comment]
Isn't this over-complicated? 
IIUC, we are trying to reuse previous `lint` rule from individual packages.
And the only thing preventing us from reusing the existing top-level `lint` script is the folder `.` specified there.
The simplest solution is to introduce a new generic rule without the the folder, so specialized rule can reuse it by calling it with one extra parameter (the folder). Sort of `generic-lint` rule or similar. This approach seems to me much more clear at the first glance, so without the need to dig into another .ts file to get understanding what the linter does.

Please note, there is already related PR: #1411

**Comment by christianvogt** [comment]
Adding a new eslint script without a path would be very flexible and solve your use case:
` "eslint": "eslint --ext .js,.jsx,.ts,.tsx --color",

Then do:
`yarn eslint packages/kubevirt`

or be more specific to test a single file:
`yarn eslint **/MyFile.tsx`

**Comment by suomiy** [comment]
yes that is much easier (although without autocompletion)

- added generic eslint script

**Comment by suomiy** [comment]
Tests passed. Can we proceed with this one?

File: frontend/package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "dev": "yarn clean && yarn plugin-stats && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
     "build": "yarn clean && yarn plugin-stats && NODE_ENV=production ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production",
     "coverage": "jest --coverage .",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx --color .",
+    "eslint": "eslint --ext .js,.jsx,.ts,.tsx --color",
+    "lint": "yarn run eslint .",
```
Comment (vojtechszocs) [inline frontend/package.json]: Nit: this can be simply `yarn eslint .` - any non-standard Yarn command is automatically treated as `yarn run <cmd>`.

Comment (suomiy) [reply frontend/package.json]: removed run

**Comment by suomiy** [comment]
> LGTM, but note that this causes the project-specific eslint script to over-shadow the actual binary ./node_modules/.bin/eslint (shouldn't be a problem, though).

Renamed to `lintme`. Let me know if somebody comes up with a better name

File: frontend/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "dev": "yarn clean && yarn plugin-stats && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
     "build": "yarn clean && yarn plugin-stats && NODE_ENV=production ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production",
     "coverage": "jest --coverage .",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx --color .",
+    "lintme": "eslint --ext .js,.jsx,.ts,.tsx --color",
```
Comment (vojtechszocs) [inline frontend/package.json]: I'd prefer to go with @christianvogt's suggestion:

> Adding a new eslint script without a path would be very flexible and solve your use case:
> `"eslint": "eslint --ext .js,.jsx,.ts,.tsx --color",`

Please replace `lintme` with `eslint`. All developers should use the same `eslint` script for unified experience.

The fact that `eslint` script in `package.json` over-shadows `./node_modules/.bin/eslint` delegation (`yarn run`) is not a problem, so no need to come up with alternative script names like `lintme`.

File: frontend/package.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
...
+    "lint": "yarn lintme .",
```
Comment (vojtechszocs) [inline frontend/package.json]: This should be `yarn eslint .`

**Comment by suomiy** [comment]
reverted back to `eslint`

**Comment by vojtechszocs** [comment]
/lgtm

cc @christianvogt

**Pull request merged**